test/test_kubeclient.rb

```diff
@@ -519,15 +519,15 @@ def test_proxy_url
                  client.proxy_url('service', 'srvname', 'srvportname', 'ns'))
 
     assert_equal('http://host:8080/api/v1/namespaces/ns/pods/srvname:srvportname/proxy',
-                 client.proxy_url('pods', 'srvname', 'srvportname', 'ns'))
+                 client.proxy_url('pod', 'srvname', 'srvportname', 'ns'))
```

Comment:
Right but using plural here is against what we say in the README as well as #116 .
We can retain support by adding a condition in proxy_url.

ps https://github.com/kubernetes/kubernetes/issues/18622 is interesting (though not really related)